### PR TITLE
Add social links on website Issue 20

### PR DESCRIPTION
**Pull request by jabhishek87**
Fixes issue #20  
Added logo for twitter, github, zulip
 update css
 added social links

### CODE REVIEW

**Comment by jabhishek87** [comment]
Screen shot 
image

**Comment by inovizz** [comment]
the build seems to be failing can you check @jabhishek87 @bhansa ?

**Comment by jabhishek87** [comment]
@inovizz I don't know why it is failing? seems like an issue with the jobs itself

**Comment by inovizz** [comment]
@jabhishek87 - ok I'll look into it. Thanks.

File: _includes/footer.html [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
-      <a href="dummy">dummy</a>
+
+    <div class="copyright-details">
+      <div class="copyright-text">Copyright 2020 Pyconf hyderabad</div>
+      <div class="copyright-logo"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png"></div>
     </div>
+
+    <div class="social-media-details">
+
+      <div>
```
Comment (bhansa) [inline _includes/footer.html]: use a class name for these divs

File: _sass/_footer.scss [hunk 2]
```diff
@@ -9,6 +9,21 @@ footer {
...
     flex-wrap: wrap;
+    .copyright-details {
+      text-align: center;
+      padding-top: 0.6875rem;
+    }
+    .social-media-details {
+      padding-top: 1.1875rem;
+      display: flex;
+      justify-content: space-between;
+      div {
```
Comment (bhansa) [inline _sass/_footer.scss]: reference the above classnames here

File: _includes/footer.html [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
+        <a href="https://github.com/HydPy/pyconfhyd2020" rel="nofollow noreferrer" target="_blank">
+          <img src="../2020/assets/images/logo/github-logo.svg" alt="github">
+        </a>
+      </div>
+
+      <div>
+        <a href="https://hydpy.zulipchat.com/" rel="nofollow noreferrer" target="_blank">
+          <img src="../2020/assets/images/logo/chat-logo.svg" alt="zulip">
```
Comment (bhansa) [inline _includes/footer.html:28]: these url references might not work if you are on speaker description page, please verify.
Click on speaker image on homepage to navigate to speaker description page.

**Review by bhansa — CHANGES_REQUESTED** [review]

**Comment by bhansa** [comment]
@jabhishek87 Added few comments above, please work on them. Rest looks good.

**Comment by bhansa** [comment]
@jabhishek87 any updates on this?

File: _includes/footer.html (hunk 1 above)
Comment (jabhishek87) [reply _includes/footer.html]: fixed

File: _sass/_footer.scss (hunk 2 above)
Comment (jabhishek87) [reply _sass/_footer.scss]: fixed

File: _includes/footer.html (hunk 3 above)
Comment (jabhishek87) [reply _includes/footer.html:28]: speaker page does not have footer ?

Comment (bhansa) [reply _includes/footer.html:28]: It does have footer.

**Review by bhansa — APPROVED** [review]

**Pull request merged**